docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -61,3 +61,5 @@ Once endpoints E1 and E2 exit slow start mode, their load balancing weight remai
 .. image:: /_static/slow_start_example.svg
    :width: 60%
    :align: center
+
+*Note* that slow start does not work with priorities, e.g. when traffic spills over across priorities there will be no progressive increase of traffic to destination endpoints, all traffic will shift at once.
```

Comment:
It's hard to say the slow start does not work with priorities. 

**I think the key problems is that the `slow start` can not work for cross-priorities.** It means if there is only one endpoint for a priority, then if the priority is choosed, the unique endpoint is choosed regardless of whether the endpoint in the slow start mode or not.

I am worry about that the current description will make the end users think that if multiple priorities is used then slow start will be invalid completely. But infact, the situation doesn't seem to be that serious.